InitNiKom/InitNiKom.c

```diff
@@ -28,7 +28,7 @@ void createDirOrDie(char *dirname) {
   BPTR lock;
   struct FileInfoBlock __aligned fib;
   
-  if(lock = Lock(dirname, ACCESS_READ)) {
+  if((lock = Lock(dirname, ACCESS_READ))) {
```

Comment:
Then again.. I guess that's true for many of the other updates as well. (e.g. %d -> %ld)